modules/brainbrowser/README.md

```diff
@@ -0,0 +1,42 @@
+# Brainbrowser
+
+## Purpose
+
+Brainbrowser allows visualization and navigation of the MINC images of a 
+candidate's imaging session.
+
+
+## Intended Users
+
+The two primary types of users are:
+- Imaging specialists doing online quality control (QC)
+- Project radiologists reporting incidental findings.
+
+
+## Scope
+
+Brainbrowser allows visualization of MINC images directly within its browser.
+The user can navigate those images with sagittal, coronal, and transverse 
+slices displayed.  In addition, a `Play` button is available to make feasible 
+the navigation through a 4-Dimensional image (such as time in the case of 
+resting state fMRI scans, and directions in the case of DWI scans). 
```

Comment:
@cmadjar 
it is called directions for DWI? please confirm!

Reply:
yes :)